x/dispensation/handler_test.go

```diff
@@ -32,3 +33,19 @@ func TestNewHandler(t *testing.T) {
 	dr := keeper.GetRecordsForNameAll(ctx, "AR1")
 	assert.Len(t, dr, recipients)
 }
+
+func TestNewHandler_CreateClaim(t *testing.T) {
+	app, ctx := test.CreateTestApp(false)
+	keeper := app.DispensationKeeper
+	handler := dispensation.NewHandler(keeper)
+	address := sdk.AccAddress(crypto.AddressHash([]byte("User1")))
+	msgClaim := types.NewMsgCreateClaim(address, types.ValidatorSubsidy)
+	res, err := handler(ctx, msgClaim)
+	require.NoError(t, err)
+	require.NotNil(t, res)
```

Comment:
Are there other assertions that can be put here?

Reply:
We are checking the using GetClaim below , so the functionality works ,the claims get saved . 
Checking the hash and verifying it by querying the blockchain should come under the integration tests I guess.